NitroxServer/Server.cs

```diff
@@ -66,12 +66,8 @@ public void Save()
                 return;
             }
 
-            // Don't overwrite config changes that users made to file
-            if (!File.Exists(serverConfig.FileName))
-            {
-                NitroxConfig.Serialize(serverConfig);
-            }
             IsSaving = true;
+            NitroxConfig.Serialize(serverConfig); // This is overwriting the config file => server has to be closed before making changes to it
```

Comment:
I guess the comment before wasn't clear that this introduces a bug and overrides the player's own config setting on close. Any command that changes settings should read the existing settings, set the value, then write it back immediately.

Reply:
"_Any command that changes settings should read the existing settings, set the value, then write it back immediately._" Unfortunately that is not the case. I will revert my changes and write a config save function.

Reply:
A config save function is not necessary I think. The `swapSerializer` can resave the config file when called. It should be transparent to the user to avoid confusion on when things are saved (should always be saved unless logged/printed otherwise).